### PR TITLE
[API-35388] Add ethics principles attestation checkbox

### PR DESCRIPTION
**Pull request by andresriveratoro**
### Description

https://jira.devops.va.gov/browse/API-35388
+ Updates to text and styles

### CODE REVIEW

File: src/containers/consumerOnboarding/components/sandbox/OAuthCcgAppInfo.tsx [hunk 1]
```diff
@@ -23,8 +23,9 @@ const OAuthCcgAppInfo: React.FC<OAuthCcgAppInfoProps> = ({
         In order to access an API that uses OAuth 2.0 Client Credentials Grant, you must provide
         your public key.{' '}
         <Link to={ccgPublicKeyUrl} target="_blank">
-          Learn how to generate a public key.
+          Learn how to generate a public key
         </Link>
+        .
```
Comment (andresriveratoro) [inline src/containers/consumerOnboarding/components/sandbox/OAuthCcgAppInfo.tsx:28]: UX request

File: src/containers/consumerOnboarding/components/productionAccessForm/OAuthCcgAppInfo.tsx [hunk 2]
```diff
@@ -17,8 +17,9 @@ const OAuthCcgAppInfo = (): JSX.Element => {
             type="button"
             className="usa-button usa-button-secondary usa-button-unstyled"
           >
-            Learn how to generate a public key.
+            Learn how to generate a public key
           </button>
+          .
```
Comment (andresriveratoro) [inline src/containers/consumerOnboarding/components/productionAccessForm/OAuthCcgAppInfo.tsx:22]: UX request

File: src/containers/consumerOnboarding/components/productionAccessForm/OAuthAcgAppInfo.tsx [hunk 3]
```diff
@@ -17,9 +17,9 @@ const OAuthAcgAppInfo = (): JSX.Element => (
       to learn more about PKCE OAuth flow.
     </div>
     <FieldSet
-      className={classNames('vads-u-margin-top--4', 'vads-u-padding-left--1p5')}
+      className="vads-u-margin-top--4"
       legend="Can your application securely hide a client secret?"
-      legendClassName="legend-label"
+      legendClassName="vads-u-font-weight--normal vads-u-font-size--base"
```
Comment (andresriveratoro) [inline src/containers/consumerOnboarding/components/productionAccessForm/OAuthAcgAppInfo.tsx:22]: UX request

File: src/containers/consumerOnboarding/validationSchema.ts [hunk 4]
```diff
@@ -15,6 +15,24 @@ const phoneRegex =
...
+  'confirmation',
+  'decision_reviews',
+  'directDepositManagement',
+  'educationBenefits',
+  'lgyGuarantyRemittance',
+  'loanReview',
+  'health',
+  'vaLetterGenerator',
+  'verification',
+];
```
Comment (gileswells) [inline src/containers/consumerOnboarding/validationSchema.ts:35]: I am not a fan of hard coding lists of which APIs require toggle fields like this. This feels like something that should be toggled, like we do for blocking of the sandbox form, within LPB. So long as a ticket is created to move source of truth for what builds this list of APIs I am ok with is for a v1.

File: test/image_snapshots/visual.cy.js/explore-api-claims-sandbox-access-1.snap.png
Comment (gileswells) [inline test/image_snapshots/visual.cy.js/explore-api-claims-sandbox-access-1.snap.png:1]: Is "Choose your auth type" enough of a sentence to require a period like we have for the "Review our terms of service" like that got the added period?

**Review by gileswells — APPROVED** [review]
Overall approved but couple items that are not blocking that can be addressed / at minimum considered or pushed over to UX for review. Do need that ticket to move the list of APIs over to LPB.

**Pull request merged**